packages/node/src/node/server/InteractionServer.ts

```diff
@@ -442,28 +454,28 @@ export class InteractionServer implements ProtocolHandler, InteractionRecipient
             throw new StatusResponseError("No attributes or events requested", StatusCode.InvalidAction);
         }
 
-        logger.debug(
-            () =>
-                `Subscribe to attributes: ${
-                    attributeRequests?.map(path => this.#node.protocol.inspectPath(path)).join(", ") ?? "none"
-                }, events: ${eventRequests?.map(path => this.#node.protocol.inspectPath(path)).join(", ") ?? "none"}`,
+        logger.debug(() =>
```

Comment:
Maybe either `via, "Subscription details"` as log message prefix, or conditionally merge with the message above?